### PR TITLE
Kbrose resuming

### PR DESCRIPTION
**Pull request by lukeforehand**

### CODE REVIEW

File: deertracker/classifier.py [hunk 1]
```diff
@@ -168,12 +168,20 @@ def train(
...
+    resume: bool = False,
 ):
     model_dir = model_dir / model_name
-    if model_dir.exists():
+    if model_dir.exists() and not resume:
         raise ValueError(
             f"{model_dir} already exists, refusing to overwrite saved models."
+            " If you wanted to resume, please set resume to True."
         )
+    if resume and not (model_dir / "ckpts").exists():
```
Comment (lukeforehand) [inline deertracker/classifier.py:179]: @kbrose should I have had to add this?

**Comment by lukeforehand** [comment]
I worked awesomely!!

File: deertracker/classifier.py (hunk 1 above)
Comment (kbrose) [reply deertracker/classifier.py:179]: Yep that looks correct. My bad!

Comment (lukeforehand) [reply deertracker/classifier.py:179]: no worries!  merged and working great!

**Comment by lukeforehand** [comment]
@kbrose I did a resume on epoch 11, and didn't reach a better "test loss for awhile, was getting worried but at epoch 221 got a new saved model!  Resume is working that exciting

**Comment by kbrose** [comment]
Nice! :tada:

**Pull request merged**